xiplot/plots/scatterplot.py

```diff
@@ -376,43 +319,10 @@ def render(
 
         return fig
 
-    @staticmethod
-    def create_layout(index, df, columns, config=dict()):
+    @classmethod
+    def create_layout(cls, index, df, columns, config=dict()):
         num_columns = get_numeric_columns(df, columns)
 
-        try:
-            if config["axes"]["x"] in (
-                "Xiplot_PCA_1",
-                "Xiplot_PCA_2",
-            ) or config["axes"]["y"] in (
-                "Xiplot_PCA_1",
-                "Xiplot_PCA_2",
-            ):
-                num_columns.append("Xiplot_PCA_1")
-                num_columns.append("Xiplot_PCA_2")
-
-        except Exception:
-            pass
-
-        jsonschema.validate(
```

Comment:
validate?

Reply:
re-added (all of them, including the heatmap one)